### PR TITLE
Update settings.yml

### PR DESCRIPTION
**Pull request by MODKILLER1001**

### CODE REVIEW

File: PlotSquared/config/settings.yml [hunk 1]
```diff
@@ -36,18 +36,18 @@ enabled-components:
   # Allow WorldEdit to be restricted to plots
   worldedit-restrictions: true
   # Allow economy to be used
-  economy: true
+  economy: false
```
Comment (imAero) [inline PlotSquared/config/settings.yml]: People might want to be able to sell plots to players.

**Comment by imAero** [comment]
I added a comment on line 39 if you'd like to take it into consideration.a

**Comment by MODKILLER1001** [comment]
@imAero did you change the sell option

**Comment by imAero** [comment]
Sell option for? I'm confused.

**Comment by MODKILLER1001** [comment]
the sell potion for plots https://gyazo.com/fc7b76730132d84a9ef43dbe89b3223b

**Comment by imAero** [comment]
I'd say just leave the config the way it is for now. It works fine as-is.

**Pull request closed without merging**